exla/c_src/exla/exla_client.cc

```diff
@@ -194,7 +186,6 @@ xla::StatusOr<ERL_NIF_TERM> ExlaExecutable::Run(ErlNifEnv* env,
       // so it's tracked and GC'ed along with other buffers that are no
       // longer in use
       if (buf->release_after_run()) {
-        EXLA_EFFECT_OR_RETURN(buf->BlockHostUntilReady());
         terms.push_back(nif::make<ExlaBuffer*>(env, buf));
```

Comment:
I guess this kinda ensures the reference is kept until the function returns (otherwise I'm almost sure it would only live throughout the `if` block). Not sure on the implication for this though.